features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/backend-service-invoker.js

```diff
@@ -128,7 +130,7 @@ var backendServiceInvoker = function () {
         header.setValue(constants.APPLICATION_JSON);
         httpMethodObject.addRequestHeader(header);
         if (IS_OAUTH_ENABLED) {
-            var accessToken = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER).accessToken;
+            var accessToken = getAccessToken();
             if (!(!accessToken.trim())) {
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/commit/3e537e2d841ca3c7946cd34e9da96f261e0ac94e1